blocks/library/more/index.js

```diff
@@ -21,12 +21,11 @@ registerBlockType( 'core/more', {
 
 	useOnce: true,
 
-	className: false,
-
 	supportHTML: false,
 
 	supports: {
 		customClassName: false,
+		generatedClassName: false,
```

Comment:
wonder if we could call this `wpClassName`

Reply:
or even just `className`

Reply:
Or `autoClassName` ? Or simply `className` ?

Reply:
`className` also sounds fine.